xbmc/cores/DllLoader/Win32DllLoader.cpp

```diff
@@ -324,6 +324,9 @@ void Win32DllLoader::OverrideImports(const std::string &dll)
 
 bool Win32DllLoader::NeedsHooking(const char *dllName)
 {
+  if (!StringUtils::EndsWithNoCase(dllName, "libdvdread.dll"))
```

Comment:
It was a guess from @Paxxi , should I better use the two you mentioned?
I have no glue, I only want this emulation far away from the binary addons I use.